contrib/platform/src/com/sun/jna/platform/win32/Advapi32Util.java

```diff
@@ -1758,13 +1758,14 @@ public EnumKey(HKEY hKey, int dwIndex) {
 	 * @return A environment block
 	 */
 	public static String getEnvironmentBlock(Map<String, String> environment) {
-		StringBuilder out = new StringBuilder();
+		StringBuilder out = new StringBuilder(environment.size() * 32 /* some guess about average name=value length*/);
```

Comment:
Personally I think this is premature optimization, since I cannot think of a reason why to call this in a tight loop.